### PR TITLE
Blacklisting RunWrapper.getRawBuild

### PR DESCRIPTION
**Pull request by jglick**
`currentBuild.rawBuild` is not in and of itself a risk in a Pipeline script, but pretty much anything you would do with the result would be, so best to nip this in the bud.

@reviewbybees

### CODE REVIEW

File: src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/blacklist [hunk 1]
```diff
@@ -22,3 +22,6 @@ staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods execute java.lang.
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods execute java.util.List
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods execute java.util.List java.lang.String[] java.io.File
 staticMethod org.codehaus.groovy.runtime.DefaultGroovyMethods execute java.util.List java.util.List java.io.File
+
+# TODO do we need a @Blacklisted annotation?
```
Comment (svanoort) [inline src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/blacklist]: That would be nice, yes.

**Comment by svanoort** [comment]
🐝

**Comment by andresrc** [comment]
:bee:

**Comment by alvarolobato** [comment]
🐝

**Comment by daniel-beck** [comment]
Does this need a compatibility warning? What happens when it's already whitelisted?

**Comment by jglick** [comment]
> What happens when it's already whitelisted?

ALL YOUR JENKINS HOME BELONG TO US

**Comment by jglick** [comment]
(well…no, but you should have known better)

**Pull request merged**